src/game/traps/teeth_trap.c

```diff
@@ -12,6 +13,19 @@ BITE_INFO Teeth2B = { 71, 10, -1718, 1 };
 BITE_INFO Teeth3A = { -23, -10, -1718, 0 };
 BITE_INFO Teeth3B = { 71, -10, -1718, 1 };
 
+static void BaddieBiteEffect(ITEM_INFO *item, BITE_INFO *bite);
+
+static void BaddieBiteEffect(ITEM_INFO *item, BITE_INFO *bite)
```

Comment:
why teeth_trap and not control which holds all the global "creature" code? Or in blood.c of which this creates and sets up?

Reply:
I'm not sure I understand your question but I moved it to this file, because it is the only usage of this function across the entire codebase.